Sources/GeneratorCLI/GeneratorCLI.swift

```diff
@@ -100,28 +100,21 @@ extension GeneratorCLI {
 
     @Option(
       help: """
-      CPU architecture of the host triple of the bundle. Defaults to a triple of the machine this generator is \
-      running on if unspecified. Available options: \(
-        Triple.Arch.allCases.map { "`\($0.rawValue)`" }.joined(separator: ", ")
-      ).
+      The host triple of the bundle. Defaults to a triple of the machine this generator is \
+      running on if unspecified.
       """
     )
-    var hostArch: Triple.Arch? = nil
+    var host: Triple? = nil
 
     @Option(
       help: """
-      CPU architecture of the target triple of the bundle. Same as the host triple CPU architecture if unspecified. \
-      Available options: \(Triple.Arch.allCases.map { "`\($0.rawValue)`" }.joined(separator: ", ")).
+      The target triple of the bundle. Same as the host triple if unspecified.
```

Comment:
This updated help string looks wrong to me, unless the SDK generator doesn't generate anything useful in its default configuration? What SDK does it generate if host and target triples are the same and you run it on macOS?

Reply:
Oh, good catch. The default target triple now depends on the recipe subcommand, how should we describe it in help 🤔

Reply:
I think the previous behaviour (when the generator was mainly for Linux SDKs) was to match the host and target CPU architecture if CPU wasn't specified.